Jenkinsfile

```diff
@@ -8,7 +8,8 @@ def genBranch(String arch) {
 				wrappedNode(label: "linux&&${arch}", cleanWorkspace: true) {
 					try {
 						checkout scm
-						sh("git clone https://github.com/docker/engine.git engine")
+						sh("git clone https://github.com/docker/engine.git")
+						sh("git -C engine checkout $branch")
```

Comment:
For some reason, it only does the `git clone`, and after that continues with the `finally` (`make ENGINE_DIR=$(pwd)/engine clean-image clean-engine`).

So for some reason, the `git clone` is failing, but no error is shown. Perhaps a permissions issue, or the `engine` directory exists (and is not empty?)